ampersand-form-view.js

```diff
@@ -183,7 +187,7 @@ module.exports = View.extend({
             delete this._startingValues;
         }
         this.handleSubmit = this.handleSubmit.bind(this);
-        this.el.addEventListener('submit', this.handleSubmit, false);
+        this.formEl.addEventListener('submit', this.handleSubmit, false);
```

Comment:
understood.

> is in cases where the root el is not the <form> tag

is this case a reality, however?  FormView is essentially a wrapper around `<form>` itself to offer a programatic I/O of FieldValues, despite FieldView instances need not needing to actually be nested inside the `<form>` heirarchy.